index.html

```diff
@@ -0,0 +1,41 @@
+---
+layout: default
+---
```

Comment:
Yes. The html files here are processed by Jekyll on github's servers before being served